generate_specs.py

```diff
@@ -187,23 +187,47 @@ def verify_one_function(func_name: str, llvm_analysis: LLVMAnalysis, out_file: P
     return success
 
 
-if __name__ == "__main__":
-    inp_file = Path(sys.argv[1])
+def _prepare_output_location(input_file_path: Path) -> Path:
+    """Return the path to the output location of the C program with generated specs.
 
```

Comment:
The output file is identical to the input file, except that it may contain extra `#include` statements.